### ISSUE TITLE
Pages with missing alt tags

### ISSUE THREAD

**Issue by MangopearUK** [MangopearUK/European-Boating-Association--Theme#403]
1. https://eba.eu.com/2015/11/new-mobile-phone-apps-for-boaters-on-the-inland-waterways/
2. https://eba.eu.com/2015/10/safety-at-sea-check-your-lifejacket/

**Comment by MangopearUK** [comment]
3. https://eba.eu.com/2015/08/donnau-danube-apps-fur-smartphones/

**Comment by MangopearUK** [comment]
4. https://eba.eu.com/2015/07/take-part-in-the-rya-kill-cord-and-kill-switch-failure-survey/

**Comment by MangopearUK** [comment]
5. https://eba.eu.com/2014/10/wind-turbine-areas-in-netherlands-part-of-the-north-sea/

**Comment by MangopearUK** [comment]
6. https://eba.eu.com/2012/02/lode-na-vode-boats-on-the-water/

**Comment by MangopearUK** [comment]
Closed by client.
